website/docs/docs/deploy/job-scheduler.md

```diff
@@ -1,33 +1,35 @@
+description: "The dbt Cloud job scheduler queues scheduled or API-triggered runs, before preparing the job to enter cloud data platform. Build observability into transformation workflows with the in-app scheduling, logging, and alerting." 
 tags: [scheduler]
 ---
 
-The Scheduler is the backbone of running jobs in dbt Cloud, bringing power and simplicity to building data pipelines in both continuous integration and production contexts. The Scheduler frees teams from having to build and maintain their own infrastructure and ensures the timeliness and reliability of data transformations.
+The job scheduler is the backbone of running jobs in dbt Cloud, bringing power and simplicity to building data pipelines in both continuous integration and production contexts. The scheduler frees teams from having to build and maintain their own infrastructure, and ensures the timeliness and reliability of data transformations.
 
-The Scheduler enables both cron-based and event-driven execution of dbt commands in the user’s data platform. Specifically, the Scheduler handles:
+The scheduler enables both cron-based and event-driven execution of dbt commands in the user’s data platform. Specifically, it handles:
 
 - Cron-based execution of dbt Cloud jobs that run on a predetermined cadence
 - Event-driven execution of dbt Cloud CI jobs triggered by pull requests to the dbt repo
 - Event-driven execution of dbt Cloud jobs triggered by API
 - Event-driven execution of dbt Cloud jobs manually triggered by a user to "Run Now"
 
-The Scheduler handles various tasks including queuing jobs, creating temporary environments to run the dbt commands required for those jobs, providing logs for debugging and remediation, and storing dbt artifacts for direct consumption/ingestion by the Metadata API. 
+The scheduler handles various tasks including queuing jobs, creating temporary environments to run the dbt commands required for those jobs, providing logs for debugging and remediation, and storing dbt artifacts for direct consumption/ingestion by the Metadata API. 
 
-**Note**: Executions performed in the dbt Cloud IDE aren't included in the Scheduler's handling. For example, dbt commands executed in the IDE won't affect the Scheduler or your run slots. 
+**Note**: Executions performed in the dbt Cloud IDE aren't included in the scheduler's handling. For example, dbt commands executed in the IDE won't affect the Scheduler or your run slots. 
 
-The Scheduler powers running dbt in staging and production environments, bringing ease and confidence to CI/CD workflows and enabling observability and governance in deploying dbt at scale. 
+The scheduler powers running dbt in staging and production environments, bringing ease and confidence to CI/CD workflows and enabling observability and governance in deploying dbt at scale. 
+
+<Lightbox src ="/img/dbt-cloud-jobs.gif" width="85%" title="An overview of a dbt Cloud job run"/>
```

Comment:
can we delete this run history image? I don't think it belongs here, and we just updated run history [here](https://docs.getdbt.com/docs/deploy/dbt-cloud-job#run-history). Let's make sure we don't lose this info.